### DISCUSSION TITLE
Un-Normalized Confusion Matrix

### DISCUSSION THREAD

**Discussion by aonsafdar** [ultralytics/yolov5#8781] [Q&A]
Hi, the confusion matrix shows normalized values for classifications made by the model. What changes do I have to make if I want to plot the confusion matrix with original values (not normalized) for classifications?

**Comment by glenn-jocher** [comment]
@aonsafdar set normalize=False
https://github.com/ultralytics/yolov5/blob/52d3a9aee1016604652898fed679e55783e264ed/utils/metrics.py#L187

**Reply by aonsafdar** [reply]
Thanks, glenn!
